### PR TITLE
feat: raw implementation of DB tests

### PR DESCRIPTION
**Pull request by stepanLav**
That PR adds integration tests for data migration from 17 to 18 model.
To run it locally - `pnpm test:migration`
It also will run on pull requests via CI

UPD:
Changed trigger to PR when destination is `main`

### CODE REVIEW

**Comment by pgolovkin** [comment]
Overall it looks good!

File: tests/integrations/migrations/migration-2.test.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    expect(watchOnlyAccountBD).toMatchObject({
+      walletId: watchOnlyWalletBD.id,
+      name: watchOnlyWalletBD.name,
+    });
+
+    expect(watchOnlyWalletBD).toMatchObject({ signingType: 'signing_wo' });
+    expect(watchOnlyWalletBD.isActive).toBe(true);
+  });
+
+  afterEach(async () => {
```
Comment (tuul-wq) [inline tests/integrations/migrations/migration-2.test.tsx:158]: Put `afterEach` after `beforeEach`

**Review by tuul-wq — APPROVED** [review]
I assume we should rename `tests` folder to `e2e` in future

File: tests/integrations/migrations/migration-2.test.tsx (hunk 1 above)
Comment (stepanLav) [reply tests/integrations/migrations/migration-2.test.tsx]: fixed it, thank you!

**Comment by stepanLav** [comment]
> I assume we should rename `tests` folder to `e2e` in future

I would prefer that structure:

project_root/
│
├── tests/
│   ├── integration/
│   │   ├── migration/
│   │   │   ├── migration.test.ts
│   │   │   └── ... (other test files)
│   │   ├── utils/
│   │   │   └── ... (other utility files)
│   │   └── ... (other directories and test files)
│   │
│   └── e2e/
│       ├── ... (e2e test files)
│       └── ... (other directories and test files)

**Pull request merged**